ts/routes/image-occlusion/add-or-update-note.svelte.ts

```diff
@@ -37,7 +37,9 @@ export const addOrUpdateNote = async function(
             backExtra,
             tags,
         });
-        showResult(mode.noteId, result, noteCount);
+        if (result.note) {
```

Comment:
(may be misunderstanding): This no longer appears to notify the user of unexpected errors